packages/sdk-react/src/components/receivables/Financing/FinanceTab/FinancedInvoicesTable/FinancedInvoicesTable.tsx

```diff
@@ -0,0 +1,285 @@
+        renderCell: ({ value }) => {
+          if (!value) {
+            return (
+              <span className="Monite-TextOverflowContainer">
+                <Typography
+                  color="text.secondary"
+                  component="span"
+                  fontSize="inherit"
+                >{t(i18n)`INV-auto`}</Typography>
+              </span>
+            );
+          }
+
+          return <span className="Monite-TextOverflowContainer">{value}</span>;
+        },
+      },
+      {
+        field: 'payer_type',
+        headerName: t(i18n)`Customer`,
+        display: 'flex',
+        width: defaultCounterpartColumnWidth,
+        renderCell: (params) => (
+          <Typography>
+            {params.row.payer_type == 'BUSINESS'
+              ? params.row.payer_business_name
+              : `${params.row.payer_first_name} ${params.row.payer_last_name}`}
+          </Typography>
+        ),
+      },
+      {
+        field: 'status',
+        headerName: t(i18n)`Status`,
+        sortable: ReceivableCursorFields.includes('status'),
+        width: 160,
+        renderCell: ({ value: status }) => {
+          return <InvoiceStatusChip status={status} />;
+        },
+      },
+      {
+        field: 'total_amount',
+        headerName: t(i18n)`Amount due`,
+        headerAlign: 'right',
+        align: 'right',
+        width: 120,
+        valueGetter: (_, row) => {
+          const value = row.repayment_schedule?.repayment_amount;
+
+          return value ? formatCurrencyToDisplay(value, row.currency) : '-';
+        },
+      },
+      {
+        field: 'repayment_schedule',
+        headerName: t(i18n)`Loan sum`,
+        headerAlign: 'right',
+        align: 'right',
+        width: 120,
+        valueGetter: (_, row) => {
+          const value = row.repayment_schedule?.repayment_principal_amount;
+
+          return value ? formatCurrencyToDisplay(value, row.currency) : '-';
+        },
+      },
+      {
+        field: 'issue_date',
+        headerName: t(i18n)`Financing date`,
+        sortable: false,
+        width: 140,
+        valueFormatter: (value) => (value ? i18n.date(value, dateFormat) : '—'),
+      },
+      {
+        field: 'due_date',
+        headerName: t(i18n)`Payment date`,
+        sortable: false,
+        width: 120,
+        valueFormatter: (value) => (value ? i18n.date(value, dateFormat) : '—'),
+      },
+    ];
+  }, [formatCurrencyToDisplay, i18n, dateFormat]);
+
+  const gridApiRef = useAutosizeGridColumns(
+    collectionData,
+    columns,
+    false,
+    // eslint-disable-next-line lingui/no-unlocalized-strings
+    'FinancedInvoicesTable'
+  );
+  if (!isLoading && collectionData?.length === 0) {
+    return (
+      <DataGridEmptyState
+        title={t(i18n)`No financed invoices yet`}
+        descriptionLine1={t(
+          i18n
+        )`Select invoices you would like to finance and send them for review.`}
+        descriptionLine2={t(i18n)`What invoices can be financed?.`}
```

Comment:
There's an extra period `.` at the end:
```suggestion
        descriptionLine2={t(i18n)`What invoices can be financed?`}
```

Also, does this sentence need a link or tooltip?

Reply:
I don't see any link but if one is needed, it can be added with another PR.